### PR TITLE
feature/TSS-787/add-new-fields-from-mve-to-data-workspace

### PR DESCRIPTION
**Pull request by osimuka**

### CODE REVIEW

File: api/barriers/serializers/data_workspace.py [hunk 1]
```diff
@@ -184,6 +185,9 @@ class DataWorkspaceSerializer(AssessmentFieldsMixin, BarrierSerializerBase):
     proposed_estimated_resolution_date = serializers.SerializerMethodField()
     proposed_estimated_resolution_date_user = serializers.SerializerMethodField()
     proposed_estimated_resolution_date_created = serializers.SerializerMethodField()
+    main_sector = serializers.SerializerMethodField()
```
Comment (chris-pettinga) [inline api/barriers/serializers/data_workspace.py:188]: Can we not use the SectorsField that Feroze created

**Review by chris-pettinga — CHANGES_REQUESTED** [review]

File: api/barriers/serializers/data_workspace.py (hunk 1 above)
Comment (osimuka) [reply api/barriers/serializers/data_workspace.py:189]: Yes, but I will be looking for the string name itself, I have adjusted the method to use `SectorField` thanks for pointing it out.
The dataworkspace is looking for a string or None in that field

Comment (chris-pettinga) [reply api/barriers/serializers/data_workspace.py:189]: got you, didn't realise this was specifically the DataWorkspaceSerializer

Comment (osimuka) [reply api/barriers/serializers/data_workspace.py:189]: No worries, but the pointers helped very well thanks

File: api/barriers/serializers/data_workspace.py [hunk 2]
```diff
@@ -478,3 +488,22 @@ def get_proposed_estimated_resolution_date_user(self, instance):
...
         )
         return f"{first_name} {last_name}" if first_name and last_name else None
+
+    def get_main_sector(self, instance) -> typing.Optional[str]:
+        main_sector = SectorField(instance=instance, required=False)
+        if main_sector.is_valid():
+            return main_sector.data["name"]
+        return None
+
+    def get_export_types(self, instance) -> typing.List[str]:
```
Comment (chris-pettinga) [inline api/barriers/serializers/data_workspace.py:498]: don't need this method if export_types is now a ExportTypes field

**Review by ferozerub — APPROVED** [review]

**Review by chris-pettinga — APPROVED** [review]

**Pull request merged**